src/itemTracker/BasicItem.tsx

```diff
@@ -31,8 +33,23 @@ export function BasicItem({
     count: number;
     /** Click callback for the main item */
     onClick: (take: boolean) => void;
+    /** Item name to transfer when dragged onto a location or region */
+    dragItemName: string;
     children?: React.ReactNode;
 } & Omit<React.HTMLProps<HTMLDivElement>, 'onClick'>) {
+    // We need to create a canvas for the image so it is a consistent width
+    const dragIcon = new Image();
+    dragIcon.src = findRepresentativeIcon(dragItemName);
+    const ctx = document.createElement("canvas").getContext("2d");
+
+    dragIcon.onload = () => {
+        const width = 108;
+        const height = dragIcon.height * (width / dragIcon.width);
+        ctx!.canvas.width = width;
+        ctx!.canvas.height = height;
+        ctx!.drawImage(dragIcon, 0, 0, width, height);
+    }
```

Comment:
Doing this every render for every item looks wasteful, maybe we can use a global cache that maps from image URL -> loaded Image and only draw to a canvas when actually dragging?